hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/SqsBackendResolver.java

```diff
@@ -33,6 +42,17 @@
     * */
     Entity.Builder entityBuilder =
         getBackendEntityBuilder(BackendType.SQS, backendURI.get(), event);
-    return Optional.of(entityBuilder.build());
+    setAttributeForFirstExistingKey(
```

Comment:
If the attribute was not set in the entity before, let's not add it to the entity